app/pages/analyst/applications.tsx

```diff
@@ -50,22 +50,26 @@ class Applications extends Component<Props> {
     return {
       variables: {
         orderByField: 'operator_name',
-        direction: 'ASC'
+        direction: 'ASC',
+        max_results: 10,
+        offset: 0
       }
     };
   }
 
   render() {
     const {query} = this.props;
+    /** maxResultsPerPage matches the value of max_results in getInitialProps().
+     *  max_results is passed to the initial query via the router
+     *  maxResultsPerPage is passed as a prop to aid in rendering the correct number of pages in the FilterableTablePagination component
+     **/
+    const maxResultsPerPage = 10;
```

Comment:
You could just call the static function above

Reply:
call it in the FilterableTablePagination component?
Something like this?
`const [max, setMax] = useState(null);

  const getMaxResultsPerPage = async () => {
    const initProps = await Applications.getInitialProps();
    setMax(initProps.variables.max_results);
  }

  useEffect(() => {
    if (!max)
      getMaxResultsPerPage();
  }, []);`

Reply:
ugh, 'insert code' option is ugly. Here's a screenshot of what I added to FilterableTablePagination to get the value out of the static async getInitialProps()
![getmax](https://user-images.githubusercontent.com/26367840/109894351-2aa3dd80-7c42-11eb-8a3c-aecb528b508e.png)

Reply:
actually I guess it should go in ApplicationListContainer since this is a re-usable component, but same idea

Reply:
it probably doesn't have to be an async function. It's only async because it's a requirement of `getInitialProps`

Reply:
without `await` it returns a promise rather than the object, which is why I wrapped it in an `async`